mobile-browser-based-version/package.json

```diff
@@ -7,7 +7,7 @@
     "prod": "vue-cli-service serve --mode production",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha --exit -r ts-node/register --file tests/setup.ts 'tests/**/*.ts'",
+    "test": "( cd server && npm start ) & env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha --exit -r ts-node/register --file tests/setup.ts 'tests/**/*.ts' && kill $! && wait",
```

Comment:
the server takes a bit of time to build (2-3s), maybe in the future it might be longer, could that be a problem? e.g. the tests start before the server finishes building.

Reply:
Right, I'm now using a script to ensure that it is started before running the tests: `with_server`